src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -5,7 +5,7 @@ import scala.math.BigDecimal
  * Numerical utilities for implementation and test cases.
  */
 object NumericUtils {
-  val EPSILON = 1.0e-6
+  val EPSILON = 1.0e-5
```

Comment:
What's up with this change? I think I've always seen `1.0e-6` in other projects.